src/utils/privileges.cpp

```diff
@@ -249,10 +254,13 @@ void verify_privileges_are_as_for_mainloop() {
 }
 
 
-void drop_root() {
+std::string drop_root(bool home_always_env) {
+	std::string home_dir = security_drop_root_from_sudo();
 	#ifdef ANTINET_linux
 	try {
-		security_drop_root_from_sudo();
```

Comment:
home_dir = security_drop_root_from_sudo();

threfore works also if  security_drop_root_from_sudo is only defined on LINUX / MAC